cargo/spec/dependabot/cargo/file_fetcher_spec.rb

```diff
@@ -123,7 +137,7 @@
         with(headers: { "Authorization" => "token token" }).
         to_return(
           status: 200,
-          body: fixture("github", "contents_cargo_lockfile.json"),
+          body: JSON.dump({ content: Base64.encode64("[toolchain]\nchannel = \"1.2.3\"") }),
```

Comment:
Oh interesting, I was wondering if this value comes in as a version string, based on [their docs](https://rust-lang.github.io/rustup/overrides.html#channel) I was thinking it might show as `"stable-year-day-month"` so it's nice to see this!